<launch>

<!-- Load joint controller configurations from YAML file to parameter server -->
<rosparam file="$(find toycar)/config/joint_states.yaml" command="load"/>

<!-- load the controllers -->
<node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
	output="screen" ns="/toycar" args="joint_state_controller
					  joint1_position_controller
					  joint2_position_controller"/>



<!-- converting joint states to TF transforms -->
<node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"
    respawn="false" output="screen">
<remap from="/joint_states" to="/toycar/joint_states" />
</node>
 
<param name = "use_gui" value = "true"/>




</launch>

